static/bidder-params/groupm.json

```diff
@@ -0,0 +1,61 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Groupm Adapter Params",
+	"description": "A schema which validates params accepted by the Groupm adapter",
+	"type": "object",
+	"properties": {
+		"publisherId": {
+			"type": "string",
+			"description": "An ID which identifies the publisher"
+		},
+		"adSlot": {
+			"type": "string",
+			"description": "An ID which identifies the ad slot"
+		},
+		"pmzoneid": {
+			"type": "string",
+			"description": "Comma separated zone id. Used im deal targeting & site section targeting. e.g drama,sport"
+		},
+		"dctr": {
+			"type": "string",
+			"description": "Deals Custom Targeting, pipe separated key-value pairs e.g key1=V1,V2,V3|key2=v1|key3=v3,v5"
+		},
```

Comment:
I noticed what Mansi commented in your docs PR https://github.com/prebid/prebid.github.io/pull/3399, about a difference in bidder params in this file vs. what you have in the docs PR. I was wondering if intended to add any variables from the docs PR to this file, or if you'll just update the docs PR to mirror what's in this groupm.json?

Reply:
@pm-isha-bharti based on your comment from the docs PR, I am assuming that you don't need to add any  parameters here, correct?

Reply:
@pm-isha-bharti Just a friendly reminder about the above. If no changes are needed in the bidder params in Prebid Server, I believe we should be able to merge this one

Reply:
@mansinahar Yes, we don't need any changes in the bidder params JSON file. I have updated the PR prebid/prebid.github.io#3399 to include PBS bidder params in the documentation